test.sh

```diff
@@ -10,7 +10,7 @@ fi
 # Order doesn't matter. Note: godep-restore is specifically left out of the
 # defaults, because we don't want to run it locally (would be too disruptive to
 # GOPATH).
-RUN=${RUN:-vet fmt migrations unit integration errcheck}
+RUN=${RUN:-vet fmt migrations unit coverage integration errcheck}
```

Comment:
@briansmith It's merged now. You should be able to cherry-pick 1e57de to resolve the slowness.